crates/sui-json-rpc-types/src/sui_event.rs

```diff
@@ -122,6 +124,18 @@ impl SuiEvent {
     }
 }
 
+impl Display for SuiEvent {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f,
+            " ┌──\n │ Event ID: {}:{}\n │ PackageID: {}\n │ Transaction Module: {}\n │ Sender: {}\n │ EventType: {}\n",
```

Comment:
How come `Event ID` changed but not `PackageID` or `EventType`?

Reply:
`PackageID` is an `ObjectID`, so no need to change, right?
`EventType` is a `StructTag` which will use its Display implementation. 
`EventID` does not have a Display Implementation, but I found that there is a `From<EventID> for String` that basically formats it like this: `format("{:?}:{}", tx_digest, event_seq)` so I used the same formatting here.